Mathlib/NumberTheory/ModularForms/JacobiTheta/OneVariable.lean

```diff
@@ -45,13 +46,12 @@ theorem jacobiTheta_S_smul (τ : ℍ) :
     jacobiTheta ↑(ModularGroup.S • τ) = (-I * τ) ^ (1 / 2 : ℂ) * jacobiTheta τ := by
   have h0 : (τ : ℂ) ≠ 0 := ne_of_apply_ne im (zero_im.symm ▸ ne_of_gt τ.2)
   simp_rw [UpperHalfPlane.modular_S_smul, jacobiTheta_eq_jacobiTheta₂]
-  conv_rhs => erw [← ofReal_zero, jacobiTheta₂_functional_equation 0 τ.2]
+  conv_rhs => erw [← ofReal_zero, jacobiTheta₂_functional_equation 0 τ]
   rw [zero_pow two_ne_zero, mul_zero, zero_div, Complex.exp_zero, mul_one, ← mul_assoc, mul_one_div]
   erw [div_self]
   rw [one_mul, UpperHalfPlane.coe_mk, inv_neg, neg_div, one_div]
-  · rfl
-  · rw [Ne, cpow_eq_zero_iff, not_and_or]
-    exact Or.inl <| mul_ne_zero (neg_ne_zero.mpr I_ne_zero) h0
+  rw [Ne, cpow_eq_zero_iff, not_and_or]
```

Comment:
Can the two `rw`s be combined?

Reply:
Not directly, because the second one is working on a different goal. I rearranged the proof a bit to clarify (getting rid of an `erw` in the process).